intel-retail/automated-self-checkout
# feat: Added an object detection python sample app and new model

## PR Checklist


- [x ] Added label to the Pull Request for easier discoverability and search
- [ x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md

## What are you changing?
Added an object detection python sample app and new model

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

0. Build python apps: make build-python-apps
1. Download new video: cd benchmark-scripts && ./download_sample_videos.sh
2. Download new models: ./download_models/downloadOVMSModels.sh
3. Start camera simulator: ./camera-simulator/camera-simulator.sh
4. Run object detection pipeline: PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
5. Observe result output files and video output. 

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
